src_new/adapters/prebid.js

```diff
@@ -38,7 +38,14 @@ function transformPBBidToOWBid(bid, kgpv){
 	theBid.setAdUrl(bid.adUrl || "");
 	theBid.setWidth(bid.width);
 	theBid.setHeight(bid.height);
-	theBid.setReceivedTime(bid.responseTimestamp);
+
+	if (theBid.getServerSideStatus() === 0) {
```

Comment:
Remove if else from here,

theBid.setReceivedTime(bid.responseTimestamp);
theBid.setServerSideResponseTime(bid.serverSideResponseTime);

Reply:
Only above two statement is required, bid.ttl is something not related to responseTime